db/migrate/20201009201827_add_bill_association_to_billing_contract.rb

```diff
@@ -0,0 +1,5 @@
+class AddBillAssociationToBillingContract < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :bills, :billing_contracts, null: false, foreign_key: true
```

Comment:
what is this for? you already have billing_contract_id in create_table migration